packages/cli/src/util/env/env-target.ts

```diff
@@ -1,22 +1,30 @@
-import { ProjectEnvTarget } from '@vercel-internals/types';
+import type { ProjectEnvTarget } from '@vercel-internals/types';
 
-function envTargets(): string[] {
-  return Object.values(ProjectEnvTarget);
-}
-
-export function getEnvTargetChoices() {
-  return Object.entries(ProjectEnvTarget).map(([key, value]) => ({
-    name: key,
-    value: value,
-  }));
-}
+export const envTargetChoices = [
+  {
+    name: 'Production',
```

Comment:
Is `name` used anywhere? Could we instead use `.toUpperCase()` when needed?

Reply:
Yes, its used in `src/commands/env/add.ts` in a prompt to select which one the user wants to use. If we turn these into an array of strings then we can generate this choices array using map and toUpperCase i guess.